manifests/01_namespace.yaml

```diff
@@ -1,4 +0,0 @@
-apiVersion: v1
```

Comment:
Why these are removed?

Reply:
They do not respect the manifest dir layout: https://github.com/operator-framework/operator-registry/tree/release-4.2#manifest-format